tests/error/linear_errors/field_copy1.err

```diff
@@ -1,7 +1,11 @@
-Guppy compilation failed. Error in file $FILE:20
+Error: Linearity violation (at $FILE:20:11)
+   | 
+18 | def foo(s: MyStruct @owned) -> tuple[qubit, qubit]:
+19 |     t = s
+20 |     return s.q, t.q
+   |            ^^^ Field `s.q` with linear type `qubit` cannot be returned ...
+   | 
+19 |     t = s
```

Comment:
i find it hard to read messages where the same line is repeated, once in the original and then once in the sub. Just a note, don't think there is anything to be done.

Reply:
Yeah, the rendering can definitely be improved further. See #608